txnsync/peer.go

```diff
@@ -373,7 +374,7 @@ func (p *Peer) selectPendingTransactions(pendingTransactions []pooldata.SignedTx
 		effectiveBloomFilters = append(effectiveBloomFilters, filterIdx)
 	}
 
-	//removedTxn := 0
+	// removedTxn := 0
```

Comment:
It's used for debugging. We should probably should figure if we want to keep it there or not. In this PR, I've just added a space so that the linter won't give me a warning about that.